cypress/elements/common.js

```diff
@@ -0,0 +1,26 @@
+import { isYearOverYear, DIMENSION_ID_PERIOD } from '@dhis2/analytics'
+
+import { clickMenuBarUpdateButton } from './menuBar'
+import { openDimension, selectYoyCategoryOption } from './layout'
+import {
+    removeAllPeriodItems,
+    selectRelativePeriods,
+} from './dimensionModal/periodDimension'
+import { clickDimensionModalUpdateButton } from './dimensionModal'
+
+export const replacePeriodItems = (visType, useAltData) => {
```

Comment:
`useAltData` is kind of an unclear arg to the function. What does it mean... Instead, I would go with defaults:

const replacePeriodItems = (visType, period)

//for non-yoy
const testPeriodType = period.type ? period.type : 'Six-months'
const testPeriod = period.name ? period.name : 'Last six-month'

//for yoy
const testPeriod = period.name ? period.name : 'Last 2 six-months'

This would also make the test more readable (`replacePeriodItems(visType, {name: 'Quarters per year'})` instead of `replacePeriodItems(visType, true))`

Actually, I might even argue that the period info should always be passed to the function instead of having defaults.

Reply:
Yeah I would like to pass it too, but I have to pass 3 props then, as you need `nonYoyPeriodName`, `nonYoyPeriodType` and `yoyPeriodName`, since yoy periods aren't called the same as regular periods. So the keep it simple I thought I'd just provide two sets of the 3 props and just alternate between them. ATM I can't think of a use-case that would need more than two sets anyway (since you can switch back and forth between them infinite times). 
I could wrap the second arg in an object like `(visType, options)` and pass in `replacePeriodItems(myVisType, { useAltData })` if that makes it more clear than just passing the string as the second arg though?